runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -28,32 +28,54 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("chatnotification")
 public interface ChatNotificationsConfig extends Config
 {
-	@ConfigItem(
+	@ConfigSection(
+		name = "Highlight Lists",
+		description = "Custom single word and regex filter lists",
 		position = 0,
-		keyName = "highlightOwnName",
-		name = "Highlight own name",
-		description = "Highlights any instance of your username in chat"
+		closedByDefault = true
```

Comment:
We usually only set this if the full panel is not long enough to show all of the settings. But here it is, so it should probably be false.

Reply:
I choose to use this since the lists can get pretty long and this ensures that on reopening the client the other settings aren't pushed too far down to be easily accessible. Similar to how ChatFilter operates.

Reply:
If this is the only thing keeping the changes from being merged I don't mind changing it. However I think having it closed by default is the more user-friendly option.